src/explore-education-statistics-frontend/src/modules/find-statistics/components/ExploreDataButton.tsx

```diff
@@ -4,14 +4,15 @@ import ButtonLink from '@frontend/components/ButtonLink';
 import { DataBlock } from '@common/services/types/blocks';
 import { logEvent } from '@frontend/services/googleAnalyticsService';
 import React from 'react';
+import VisuallyHidden from '@common/components/VisuallyHidden';
 
 interface Props {
   block: DataBlock;
+  visuallyHiddenText?: string;
```

Comment:
As mentioned in another PR - this could be shortened to `hiddenText` as the 'visually' part is a bit verbose

Reply:
I've merged some PRs that include `visuallyHiddenText` as the prop name so I'll have a look around & change them to `hiddenText` as part of this PR